.github/workflows/build-release.yml

```diff
@@ -0,0 +1,250 @@
+    needs: create-issue
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+      - run: npm run compile
+      - uses: actions/github-script@v6
+        if: ${{ success() }}
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: ${{ needs.create-issue.outputs.issue_number }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `:heavy_check_mark: Compile test passed`
+            })
+
+  # Same job with check-pr-test.yml
+  check-test:
+    needs: create-issue
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: npm install
+      - run: sudo apt install xvfb
+      - run: DISPLAY=:44 xvfb-run --server-num 44 npm test ci
+      - uses: actions/github-script@v6
+        if: ${{ success() }}
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: ${{ needs.create-issue.outputs.issue_number }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: `:heavy_check_mark: Unit test passed`
+            })
+
+  # Same job with publish-extension.yml
+  create-vsce-package:
+    needs:
+      - post-license-result
+      - check-copyright
+      - check-compile
+      - check-test
+    runs-on: ubuntu-latest
+    outputs:
+      package_name: ${{ steps.search_package.outputs.package_name }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: Install node packages
+        run: |
+          npm install
+      - name: Install vsce tool
+        run: |
+          npm install -g vsce
+      - name: Build vsix package
+        run: |
+          vsce package ${{ github.event.inputs.tag }}
+      # TODO Enable publish job
+      - name: Publish vsix package
+        if: ${{ false }}
+        run: |
+          vsce publish
+      - id: search_package
+        run: |
+          fname=$(find . -name "*.vsix" | xargs -I {} basename {})
+          echo "::set-output name=package_name::$fname"
+      - name: Upload vsix package
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ steps.search_package.outputs.package_name }}
+          path: ./${{ steps.search_package.outputs.package_name }}
+
+  release:
+    needs: create-vsce-package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v3
+        with:
+          name: ${{ needs.create-vsce-package.outputs.package_name }}
+      - name: Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Did we add this in the repository setting?

Reply:
No. This token is automatically created. :)
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret